rpcs3/rpcs3qt/tooltips.h

```diff
@@ -170,7 +170,11 @@ class Tooltips : public QObject
 
 		// network
 
-		const QString net_status = tr("Leave as disconnected unless you're debugging.\nRPCS3 has no online support.");
+		const QString net_status = tr("If set to online, RPCS3 will allow programs to use your internet connection.");
```

Comment:
```suggestion
		const QString net_status = tr("If set to Connected, RPCS3 will allow programs to use your internet connection.");
```
Options are Connected/Disconnected